components/eamxx/src/share/io/scorpio_input.hpp

```diff
@@ -2,6 +2,7 @@
 #define SCREAM_SCORPIO_INPUT_HPP
 
 #include "share/io/scream_scorpio_interface.hpp"
+#include "share/io/scream_io_utils.hpp"
```

Comment:
I don't think this include is really needed. If it's needed in the cpp, you can just include it in the cpp.

Reply:
ok, will remove it (I think it must have been added while I was modifying the code)